acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -215,13 +217,29 @@ private String createPantheonContainer(final NodeConfiguration config) {
               "--host-whitelist",
               "*",
               "--rpc-http-enabled",
-              "--rpc-ws-enabled");
+              "--rpc-ws-enabled",
+              "--rpc-http-apis",
+              "ETH,NET,WEB3,IBFT,MINER,ADMIN,EEA");
 
       config
           .getCors()
           .ifPresent(
               cors -> commandLineItems.addAll(Lists.newArrayList("--rpc-http-cors-origins", cors)));
 
+      if (config.isPrivacyEnabled()) {
```

Comment:
why not always have privacy enabled? Or do we deliberately do tests when it is not enabled to determine correct error message? [If so - we shouldn't check the error, simply that it didn't work - we don't want to test Pantheon capability here]

Reply:
don't care about the case when privacy isn't enabled for private transactions. that is getting close to testing pantheon.
gone for just always enabling privacy a default privacy public key as that's all we need for our test.